assets/styles/_print.scss

```diff
@@ -0,0 +1,84 @@
+@media print {
+  *,
+  *:before,
+  *:after {
+    background: transparent !important;
+    color: #000 !important;
+    /* Black prints faster */
+    box-shadow: none !important;
+    text-shadow: none !important;
+  }
+
+  [is='flynt-navigation-main'],
+  [is='flynt-navigation-burger'],
+  [is='flynt-navigation-footer'],
+  [is="flynt-block-cookie-notice"],
+  .button,
+  footer {
+    display: none !important;
+  }
+
+  a {
```

Comment:
I noticed this one is not having the `:visited` state that the HTML5 Boilerplate suggests. Why?

Reply:
I thought that this rule for the <a> tag will apply to every <a> tag on the page, if `a:visited ` will get underlined, it will anyways get underlined, cause it is an <a> tag. But obviously my thoughts were wrong due to your question.